plugins/stats.js

```diff
@@ -24,11 +24,15 @@ exports.register = (server, options, next) => {
                 schema: schema.api.stats
             }
         },
-        handler: (request, reply) =>
+        handler: async (request, reply) => {
+            const executorStatsresponse = await executor.stats({ token: '' });
+            const executorStats = executorStatsresponse.body;
```

Comment:
what is this change for?

Reply:
stats api was broken after the queue service change, this change should give the correct stats result of executor and scm

Reply:
oh so this is separate

Reply:
does this works in a setup where `queue-service` is not used ?